.github/workflows/docs.yaml

```diff
@@ -25,6 +25,8 @@ jobs:
       uses: ./.github/actions/setup-ubuntu-latest-xvfb
     - name: install pandoc ubuntu
       run: sudo apt install pandoc
+    - name: install libhdf5-dev ubuntu
+      run: sudo apt install libhdf5-dev
```

Comment:
so docs need only hdf5 while the tests need a few more packages for dealing with XML? interesting

Reply:
Yes some of the awg tests uses lxml which needs these extra packages to be build from source